Source/WebKit/UIProcess/win/PageClientImpl.cpp

```diff
@@ -232,6 +240,63 @@ void PageClientImpl::preferencesDidChange()
     notImplemented();
 }
 
+bool PageClientImpl::handleRunOpenPanel(WebPageProxy*, WebFrameProxy*, const FrameInfoData&, API::OpenPanelParameters* parameters, WebOpenPanelResultListenerProxy* listener)
+{
+    ASSERT(parameters);
+    ASSERT(listener);
+
+    HWND viewWindow = viewWidget();
+    if (!IsWindow(viewWindow))
+        return false;
+
+    bool isAllowMultipleFiles = parameters->allowMultipleFiles();
+    Vector<wchar_t> fileBuf(isAllowMultipleFiles ? maxFilePathsListSize : MAX_PATH);
```

Comment:
You can't use `buf`. https://webkit.org/code-style-guidelines/#names-full-words
`fileBuf` → `fileBuffer`.

Reply:
Fixed at https://github.com/WebKit/WebKit/commit/70ce9358458f9401040946ef3392f2ae55d6fa3e